hack/manifests/testing/kustomization.yaml

```diff
@@ -14,6 +14,7 @@ patches:
 
 images:
 - name: quay.io/parodos-dev/workflow-service:main
+  newName: quay.io/parodos-dev/integration-service
```

Comment:
Is there a need to create this repository on quay?

Reply:
it is only a tag. it is never pushed to quay

Reply:
should main added here, no?

Reply:
@eloycoto what do u mean?

Reply:
Should be like this:

```
- name: quay.io/parodos-dev/workflow-service:main
  newTag: test
- newName: quay.io/parodos-dev/integration-service:main
  newTag: test
- name: quay.io/parodos-dev/notification-service:main
  newTag: test
```

Reply:
no. this is how it should be. and it works